tests/acceptance/features/lib/FilesPageElement/SharingDialogElement/EditPublicLinkPopup.php

```diff
@@ -334,4 +335,20 @@ public function waitTillPageIsLoaded(
 		$popupElement = $this->waitTillXpathIsVisible($xpath, $timeout_msec);
 		$this->setElement($popupElement);
 	}
+
+	/**
+	 *
+	 * @return $errorMessage
+	 * @throws ElementNotFoundException
+	 */
+	public function getErrorMessage() {
+		$errorMessage = $this->popupElement->find("xpath", $this->expirationDateRequiredErrorMessageXpath)->getText();
```

Comment:
You should get the element and then, check if element is there. `getText` will return text or fail here.
```suggestion 
		$errorMessageElement = $this->popupElement->find("xpath", $this->expirationDateRequiredErrorMessageXpath);
```
Then, extract message using:
```suggestion
		$errorMessage = $errorMessageElement->getText();
```